src/ui/ViewModels/CourseEnrichmentViewModel.cs

```diff
@@ -1,14 +1,49 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using GovUk.Education.ManageCourses.Ui.ViewModels.Enums;
 
 namespace GovUk.Education.ManageCourses.Ui.ViewModels
 {
     public class CourseEnrichmentViewModel
```

Comment:
some of this file has trailing whitespace, reformat it

minor

Reply:
👍